barcodes_generator_abstract/__manifest__.py

```diff
@@ -27,5 +27,5 @@
     'demo': [
         'demo/res_users.xml',
     ],
-    'external_dependencies': {'python': ['barcode']},
+    'external_dependencies': {'python': ['viivakoodi']},
```

Comment:
the code checking external_dependencies in Odoo expects a module or package name here as it tries to import the identifier => you are not supposed to provide the name of the pip package providing the module. Here viivakoodi provides a barcode module => your change cannot work.

I'm closing the PR. Please reopen if you think I'm wrong.